src/utils.rs

```diff
@@ -603,6 +603,20 @@ pub(crate) fn unicode_str_width(s: &str) -> usize {
     s.width()
 }
 
+pub fn get_skip_macro_names(attrs: &[ast::Attribute]) -> Vec<String> {
+    let mut skip_macro_names = vec![];
+    for attr in attrs {
+        if let Some(list) = attr.meta().unwrap().meta_item_list() {
```

Comment:
You can call `meta_item_list()` on `ast::Attribute` directly, thus avoiding `unwrap` here.

Reply:
Oh I didn't know. Thank you!